app/models/user.rb

```diff
@@ -33,6 +33,7 @@ def field_order
                         length: { maximum: 24 }
   validates :gender, inclusion: { in: %w(female male) }, allow_nil: true
   validates :age_range, inclusion: { in: VALID_AGE_RANGES }, allow_nil: true
+  validates :contact_number, format: { with: /\A0[12379]\d{9}\z/, message: 'is an invalid UK number', if: 'contact_number.present?'}, allow_blank: true
```

Comment:
Yep looks good

From: Jones Agyemang
Reply-To: moneyadviceservice/frontend
Date: Friday, 31 July 2015 11:57
To: moneyadviceservice/frontend
Cc: Phil Lee
Subject: Re: [frontend] TP-6556: add contact number field to sign up page (#1155)


In app/models/user.rb<https://github.com/moneyadviceservice/frontend/pull/1155#discussion_r35964188>:

> @@ -33,6 +33,7 @@ def field_order
>                          length: { maximum: 24 }
>    validates :gender, inclusion: { in: %w(female male) }, allow_nil: true
>    validates :age_range, inclusion: { in: VALID_AGE_RANGES }, allow_nil: true
> +  validates :contact_number, format: { with: /\A0[12379]\d{9}\z/, message: 'is an invalid UK number', if: 'contact_number.present?'}, allow_blank: true


Hi @asmega<https://github.com/asmega> the translation has been added for English and Welsh. I will squash it if you are happy with it.

70eb945<https://github.com/moneyadviceservice/frontend/commit/70eb9455bb625c348d0b4e088cb0982eab6c47b1>

—
Reply to this email directly or view it on GitHub<https://github.com/moneyadviceservice/frontend/pull/1155/files#r35964188>.
This communication and any attachments contain information which is confidential and may be subject to legal privilege. It is for intended recipients only. If you are not the intended recipient you must not copy, distribute, publish, rely on or otherwise use it without our consent. If you have received this email in error please notify postmaster@moneyadviceservice.org.uk immediately and delete the email from your computer. This email is not intended to nor should it be taken to create any legal relations or contractual relationships. This email has originated from The Money Advice Service, 5th Floor, Holborn Centre, 120 Holborn London EC1N 2TD United Kingdom, Registered as a Limited Company in England and Wales No. 07172704 Registered Office as above.